docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -309,6 +509,34 @@ sdlManager.sendRPC(buttonPress);
 ```
 !@
 
+You should wrap any setting data RPC in the handler of the `SDLGetInteriorVehicleDataConsent` request.
```

Comment:
I'm confused why this is here. This has nothing to do with button presses and we've already discussed consent and setting data. Can you just put a NOTE in `Setting Module Data` that when connected to v6.0+ systems they should always get consent before attempting to set module data?

Reply:
deleted section and added a note to `Setting Module Data`